content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/015-instantiate-prisma-client.mdx

```diff
@@ -51,7 +51,7 @@ The reason for this is that each instance of `PrismaClient` manages a [database
 2. Each query engine creates a [connection pool](connection-pool) with a default pool size of `num_physical_cpus * 2 + 1`.
 3. Too many connections may start to **slow down your database** and eventually lead to the following error:
 
-    ```
-    Error in connector: Error querying the database: db error: FATAL: sorry, too many clients already
-        at PrismaClientFetcher.request
-    ```
+   ```
+   Error in connector: Error querying the database: db error: FATAL: sorry, too many clients already
+       at PrismaClientFetcher.request
+   ```
```

Comment:
3 spaces indentation for code block inside list here - need to check manually if this renders correctly.

(Related https://github.com/prisma/docs/pull/1995/files#r665264673)

Reply:
Looks ok

<img width="767" alt="image" src="https://user-images.githubusercontent.com/4734297/124760851-97114f00-df4a-11eb-9192-207526bc9c6e.png">